packages/context-menu/src/vaadin-context-menu-overlay.js

```diff
@@ -3,24 +3,44 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { Overlay } from '@vaadin/overlay/src/vaadin-overlay.js';
-import { registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
+import { OverlayMixin } from '@vaadin/overlay/src/vaadin-overlay-mixin.js';
+import { overlayStyles } from '@vaadin/overlay/src/vaadin-overlay-styles.js';
+import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { MenuOverlayMixin } from './vaadin-menu-overlay-mixin.js';
 import { styles } from './vaadin-menu-overlay-styles.js';
 
-registerStyles('vaadin-context-menu-overlay', styles, { moduleId: 'vaadin-context-menu-overlay-styles' });
+registerStyles('vaadin-context-menu-overlay', [overlayStyles, styles], {
+  moduleId: 'vaadin-context-menu-overlay-styles',
+});
 
 /**
  * An element used internally by `<vaadin-context-menu>`. Not intended to be used separately.
  *
- * @extends Overlay
+ * @extends HTMLElement
+ * @mixes DirMixin
  * @mixes MenuOverlayMixin
+ * @mixes OverlayMixin
+ * @mixes PositionMixin
```

Comment:
We did not mention `PositionMixin` in the other PRs that I reviewed (confirm dialog, CRUD). Is it necessary here, or should it be added to the other elements as well?

Reply:
Actually, it isn't needed here, because `MenuOverlayMixin` already applies it. Removed.

I will check other overlay-based components and add it where it's missing.